server/controllers/booking/booking.controller.js

```diff
@@ -1,13 +1,39 @@
 import Booking from "../../models/booking";
+import car from "../../models/car";
+import User from "../../models/user";
+
+import moment from 'moment';
 
 const createBooking = function (req, res) {
-  const startsAt= req.body.startsAt;
-  const endsAt = req.body.endsAt;
+  const startAt = moment(req.body.startAt).format("YYYY-mm-ddTHH:MM:ss");
+  const endAt = moment(req.body.endAt).format("YYYY-mm-ddTHH:MM:ss");
+  const carid = req.body.car;
+  const userid = req.body.userid;
+
+  User.find({'_id': userid})
+  .exec((usrerr, selecteduser) => {
+    if (usrerr) {
+      res.status(500).send(usrerr);
+    } else if (!selecteduser) {
+      res.status(404).send();
+    } else {
+
+  car.find({'_id': carid})
+  .exec((carerr, vehicle) => {
```

Comment:
can we use camel case for variables ie carErr instead of carerr

Reply:
sure